lib/bdev/nvme/blockdev_nvme.c

```diff
@@ -357,6 +359,9 @@ blockdev_nvme_dump_config_json(struct spdk_bdev *bdev, struct spdk_json_write_ct
 				   nvme_dev->pci_addr.bus, nvme_dev->pci_addr.dev,
 				   nvme_dev->pci_addr.func);
 
+	spdk_json_write_name(w, "ctrlr_data");
```

Comment:
Since this is a brand new interface, we can get away with changing the structure around like this, since likely no one is using it yet, but it would be best to keep the naming and layout consistent unless there is a really strong reason to change it.